docs/tutorials/e2e_deprected.md

```diff
@@ -0,0 +1,92 @@
+
+:::info
+
+With the Minimum Tractus-X Dataspace (MXD), IT departments can set up their own data space locally to perform a data exchange between two parties (Alice and Bob in our example). The MXD can be used as a sandbox for testing and exploring the EDC.
+
+:::
+
+## Target Audience
+
+This tutorial is built for employees of the **IT department who operates the IT stack of an adopter (data provider/consumer)**. It shows how to operate the various Catena-X components in combination with each other. It also explains the basic connection with the core services (e.g. Keycloak and MIW), as provided by an operating company (e.g. first Catena-X Operating Company Confinity-X).
+
+:::note
+
+The related MXD doesn`t include all of the services, which are provided right now by an operating company. As the name says, right now its just a minmal set of services to learn and get in touch with the ecosystem dataspace.
+
+:::
+
+This tutorial is **not** designed for the following purposes:
+
+- It is not designed for App Providers or Service Providers.
+- It is not designed for adopters who only want to buy a business application or use fully managed services.
+- It will not show you how to setup further develop Catena-X components.
+
+:::tip
+
+Please have a look into the the [Catena-X association homepage](<https://www.catena-x.net>) information for Application and Service Providers or the [Community](<https://eclipse-tractusx.github.io/community>) page of Tractus-X in case your are more interested in topics like App Providers and Service Providers, or if you want to participate in a business application.
+
+:::
+
+## Goal of this tutorial
+
+This tutorial shall enable the IT department of an adopter (data provider/consumer) to operate the minimal Catena-X stack that is required to provide or consume data without business apps by understanding and implementing.
+
+1. The separate technical components
+2. Interfaces between components run at the adopter
+3. Connection into the wider data space
+
+During this tutorial you will get to know the perspective of a data provider as well as a data consumer and deploy the typical minimal components required.
+
+:::info
+
+In most use-cases and participation scenarios you will later use a business app for data consumption. This tutorial will nevertheless show you how the basic principles of data discovery and exchange behind the applications work.
+
+:::
+
+## Skills Required
+
+No preliminary knowledge about Catena-X is required.
+
+Technical knowledge (Docker, Kubernetes, Helm, Terraform) to deploy components onto a kubernetes cluster and interacting with them is advised. A detailed list of tools and their supported versions is given in the [prerequisites](./e2e/prerequisites/prerequisites.md).
```

Comment:
If this should stay at least for a bit, then at least the links have to be adapted. 
I would still prefer, that this PR does delete the old and add the new on one go.